packages/components/src/components.ts

```diff
@@ -22,6 +22,10 @@ import HdsIconTile from './components/hds/icon-tile/index.ts';
 import HdsInteractive from './components/hds/interactive/index.ts';
 import HdsLinkInline from './components/hds/link/inline.ts';
 import HdsLinkStandalone from './components/hds/link/standalone.ts';
+import HdsModal from './components/hds/modal/index.ts';
+import HdsModalBodyComponent from './components/hds/modal/body.ts';
```

Comment:
This being sub component, consumers will never import this directly, this file should only hold top level components and not contextual once

Reply:
Here's an example of the `Flyout::Body` sub-component (equivalent) being used directly: https://github.com/hashicorp/cloud-ui/blob/226302069218edb3d7e2fc542fc6aad0746a9000/engines/waypoint/addon/components/preview-pane.hbs#L17